app/controllers/requests_controller.rb

```diff
@@ -17,7 +17,7 @@ def new
   end
 
   def current_request
```

Comment:
In order to render the messages on the `requests/new.html.erb`, the `RequestsController` needs to implement `current_request`. @jkeck, did you have a deeper motivation to not implement it here?